.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/cesaredecal/workspace/Environments/howdoi/bin/python"
```

Comment:
You probably shouldn't check this in and instead add it to your .gitignore.

Reply:
Thank you! I forgot

Reply:
Np.